core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.controller.ts

```diff
@@ -99,34 +121,113 @@ angular.module('oppia').controller('TranslationModalController', [
       $scope.activeWrittenTranslation.html = '';
     };
 
+    $scope.hasTranslationErrors = function(data, behaviour) {
+      return data.some(state => state === behaviour);
+    };
+
+    $scope.findImgAttributes = function(elements, type) {
+      var attributes = [];
+      const textWrapperLength = 6;
+      attributes = Array.prototype.map.call(elements, function(element) {
```

Comment:
Prefer to use `Array.from` with `.forEach()` here and below in `findAvailableElements`: https://github.com/oppia/oppia/pull/11838#discussion_r587688810. Please make sure to address all parts of each review comment.

Reply:
Done